students/morgan/dnd_project/encounter_builder/data_models/monster_data.py

```diff
@@ -0,0 +1,146 @@
+import random
+import math
+
+class Monster():
+
+    def __init__(self, cr, stat_list):
+        self.cr = cr
+        self.proficiency = stat_list[0]
+        self.ac = stat_list[1]
+        self.hp_min = stat_list[2]
+        self.hp_max = stat_list[3]
+        self.attack_bonus = stat_list[4]
+        self.damage_rnd_min = stat_list[5]
+        self.damage_rnd_max = stat_list[6]
+        self.save_dc = stat_list[7]
+        self.hp = random.randint(self.hp_min, self.hp_max)
+        self.damage_rnd = random.randint(self.damage_rnd_min, self.damage_rnd_max)
+        self.str = None
+        self.dex = None
+        self.con = None
+        self.int = None
+        self.wis = None
+        self.cha = None
+        self.size = None
+        self.multi_attack = None
+        self.dmg_per_attack = None
+
+    # @classmethod
+    def generate_stats(cls):
+        sizes = ['tiny', 'small', 'medium', 'large', 'huge', 'gargantuan']
+        stat_focus = ['str', 'dex']
+        # print('cr', cls.cr)
+
+        try:
+            cr = float(cls.cr)
+        except ValueError:
+            cr = 0
+
+
+        if 0 <= cr <= 1:
+            cls.size = sizes[random.randrange(0, 1+1)]
+            this_mon_focus = stat_focus[1]
+        elif 1 < cr <= 5:
+            cls.size = sizes[random.randrange(1, 3+1 )]
+            this_mon_focus = stat_focus[random.randrange(0,1+1)]
+        elif 6 <= cr <= 10:
```

Comment:
where does 5.5 go??? if they might be floats, you want to do what you did above : <= and < the same value, so you won't miss any.